backup-stores/s3/src/test/java/io/camunda/zeebe/backup/s3/LocalstackBackupStoreIT.java

```diff
@@ -20,21 +21,40 @@
 
 @Testcontainers
 final class LocalstackBackupStoreIT implements S3BackupStoreTests {
+  private static final String BUCKET_NAME = RandomStringUtils.randomAlphabetic(10).toLowerCase();
 
   @Container
   private static final LocalStackContainer S3 =
       new LocalStackContainer(DockerImageName.parse("localstack/localstack"))
           .withServices(Service.S3);
 
-  private static S3AsyncClient client;
+  private S3AsyncClient client;
   private S3BackupStore store;
   private S3BackupConfig config;
 
+  @BeforeAll
+  static void setupBucket() {
+    final var config =
+        new Builder()
+            .withBucketName(BUCKET_NAME)
+            .withEndpoint(S3.getEndpointOverride(Service.S3).toString())
+            .withRegion(S3.getRegion())
+            .withCredentials(S3.getAccessKey(), S3.getSecretKey())
+            .withApiCallTimeout(null)
+            .forcePathStyleAccess(false)
+            .withCompressionAlgorithm(null)
```

Comment:
```suggestion       
```
These are not required for just creating the buckets. Also these are the default values.